tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2880,6 +2880,7 @@ describe('scalar card', () => {
         runToSeries
       );
       store.overrideSelector(getMetricsRangeSelectionEnabled, true);
+      store.overrideSelector(getMetricsCardRangeSelectionEnabled, true);
```

Comment:
These test now actually use the getColumnHeadersForCard selector and override the selectors which that selector uses. I kind of like this but is not a pattern we use often. Can we at least add a comment making note of this?